bluebottle/utils/views.py

```diff
@@ -1,6 +1,6 @@
-from builtins import object
 import mimetypes
 import os
+from builtins import object
```

Comment:
Instead of importing this class, just remove the `(object)` from the class definition.